rpc: fix getpeerinfo ping duration unit docs

0xB10C opened the pull request:
The docs have been incorrect since a3789c700b5a43efd4b366b4241ae840d63f2349 (released in v25; master since Sept. 2022). Noticed while setting up monitoring using getpeerinfo.

https://github.com/bitcoin/bitcoin/blob/0cb1ed2b7c634754f4ad0a0083d7cf40989d7f48/src/rpc/net.cpp#L249-L257

0xB10C committed:
rpc: fix getpeerinfo ping duration unit docs

The getpeerinfo docs incorrectly specified the ping durations as
milliseconds. This was incorrectly changed in a3789c700b5a43efd4b366b4241ae840d63f2349
(released in v25; master since Sept. 2022). The correct duration unit
is seconds.

Also, remove the documentation of the getpeerinfo RPC response from the
ping RPC since it's incomplete. Better to just reference the getpeerinfo
RPC and it's documenation for this.

jonatack reviewed (commented):
ACK 12df1a72831bf950c5d625b80c6fd6ef15b4117d

jonatack reviewed (commented):
ACK 12df1a72831bf950c5d625b80c6fd6ef15b4117d

(edit: oops, internet outage caused a duplicate ACK, so I "hid" this review)

janb84 reviewed (commented):
NIT; Please correct the commit message to better describe the change. 

The description "This has been incorrect since a3789c700b5a43efd4b366b4241ae840d63f2349
(v25; master since Sept. 2022). Noticed while setting up monitoring
using getpeerinfo." isn't descriptive of the changes in the commit.

"Commit messages should be helpful to people reading your code in the future, so explain the reasoning for your decisions.
[commit messages](https://github.com/bitcoin/bitcoin/blob/master/CONTRIBUTING.md#committing-patches)

0xB10C commented:
> Would it make sense to add "minping" here? e.g. `getpeerinfo (pingtime, minping, and pingwait fields)`

IMO it makes more sense to remove pingtime and min_ping from the ping docs. Just having `Requests that a ping be sent to all other nodes, to measure ping time. Results provided in getpeerinfo.` should be enough and ensures that it doesn't become outdated somewhere down the line?

jonatack commented:
> IMO it makes more sense to remove pingtime and min_ping from the ping docs

Indeed, that is better. Fewer ducks to keep in a row.

0xB10C commented:
- removed the `(s)` unit from the docs: https://github.com/bitcoin/bitcoin/pull/33133#discussion_r2251809033
- removed the `getpeerinfo` field documentation from the `ping` RPC: https://github.com/bitcoin/bitcoin/pull/33133#issuecomment-3152076777
- added a bit more details to the commit message: https://github.com/bitcoin/bitcoin/pull/33133#pullrequestreview-3085116958

jonatack reviewed (commented):
ACK 1252eeb997df2eb12c33d92eb1a5c9d6643a67ff

theStack reviewed (approved):
ACK 1252eeb997df2eb12c33d92eb1a5c9d6643a67ff

luke-jr commented:
utACK 1252eeb997df2eb12c33d92eb1a5c9d6643a67ff

janb84 reviewed (commented):
ACK 1252eeb997df2eb12c33d92eb1a5c9d6643a67ff

Pr changes getpeerinfo documentation to return the correct unit base (seconds instead of milliseconds) 

- Tested ✅
- Code review ✅

(Thanks for addressing the commit message !)

fanquake commented:
Backported to `29.x` in #33074.

fanquake commented:
Backported to `28.x` in #33143.
